build.gradle

```diff
@@ -29,7 +29,6 @@ dependencies {
     compile group: 'com.amazonaws', name: 'aws-java-sdk-qldb', version: '1.12.134'
     compile group: 'com.amazonaws', name: 'amazon-kinesis-client', version: '1.14.7'
 
-    compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-ion', version: '2.13.1'
-    compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-cbor', version: '2.13.1'
+    compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-ion', version: '2.12.6'
```

Comment:
Why do we downgrade?

Reply:
The latest version `2.13.1` will throw error when deserializing JournalBlock while the last latest version `2.12.6` works fine. Not sure which changes in `2.13.x` of the library caused the problem so downgrade to `2.12.x` version